format.c

```diff
@@ -27,24 +36,37 @@ fd_strplcpy(char* restrict dst, const char* src, size_t size) {
 
 static char*
 fd_strpcat(char* restrict dst, struct FdStr src) {
+#ifdef _MSC_VER
+    unsigned lim = src.sz;
```

Comment:
Unconditionally set `lim = 16` for non-GNU compilers and shrink the size of the `#ifdef`-ed region.